core/query.py

```diff
@@ -187,7 +184,7 @@ def _in_range():
 
     if not now_time:
         now_time = timezone.now()
-    return _in_range() & _active_provider()
+    return _in_range() & current_provider()
```

Comment:
I believe this is actually overuse of the `&` operator to create a `query & query` as the result returned.